projects/graph/graph.py

```diff
@@ -17,43 +17,82 @@ def add_vertex(self, vertex):
     def add_edge(self, v1, v2):
         if v1 not in self.vertices:
             self.add_vertex(v1)
-        
+
         if v2 not in self.vertices:
             self.add_vertex(v2)
-        
+
         self.vertices[v1].add(v2)
-        
+
         self.vertices[v2].add(v1)
 
     def bft(self, starting_vertex):
-        """
-        Print each vertex in breadth-first order
-        beginning from starting_vertex.
-        """
+        queue = Queue()
+
+        visited = set()
+
+        while queue.size():
+            node = queue.dequeue()
+
+            if not node in visited:
+                visited.add(node)
+
+                for edge in self.vertices[node]:
+                    queue.enqueue(edge)
         pass  # TODO
 
     def dft(self, starting_vertex):
-        """
-        Print each vertex in depth-first order
-        beginning from starting_vertex.
-        """
-        pass  # TODO
+        stack = Stack()
 
-    def dft_recursive(self, starting_vertex):
-        """
-        Print each vertex in depth-first order
```

Comment:
Be sure to print each vertex 👍 , great usage of comments throughout `DFT` and `BFS`, your future self will thank you for this when it comes time to brush up on this for an interview.